server/routes/kanban.js

```diff
@@ -1,5 +1,72 @@
 const express = require('express');
 const router = express.Router();
-const knex = require('../database/knex');
+const Card = require('../database/models/Card')
+
+function isAuthenticated(req, res, next) {
+  if (req.isAuthenticated()) { next(); }
+  else {
+    res.json({ success: false, error: 'not authenticated' });
```

Comment:
If we're rejecting a request based on authentication we want to return a `401` HTTP status code in addition to any response.